tests/testdata/fmt/with_config/subdir/a.ts

```diff
@@ -22,15 +22,11 @@ Deno.test(
 				.statusText,
 		)
 		const u8a =
-			new Uint8Array(
-				await response
-					.arrayBuffer(),
-			)
+			await response
+				.bytes()
 		const u8a1 =
-			new Uint8Array(
-				await response1
-					.arrayBuffer(),
-			)
+			await response1
+				.bytes()
```

Comment:
Nitpick: move these to be single line statements. `deno fmt` preserves line-breaks

Reply:
FYI, this test requires the code to look like this.